cluster.go

```diff
@@ -828,15 +828,21 @@ func askingMulti(cc conn, ctx context.Context, multi []Completed) *redisresults
 }
 
 func askingMultiCache(cc conn, ctx context.Context, multi []CacheableTTL) *redisresults {
-	commands := make([]Completed, 0, len(multi)*6)
+	commands := make([]Completed, 0, len(multi)*7)
 	for _, cmd := range multi {
 		ck, _ := cmds.CacheKey(cmd.Cmd)
-		commands = append(commands, cmds.OptInCmd, cmds.MultiCmd, cmds.AskingCmd, cmds.NewCompleted([]string{"PTTL", ck}), Completed(cmd.Cmd), cmds.ExecCmd)
+		commands = append(commands, cmds.OptInCmd, cmds.MultiCmd, cmds.AskingCmd, cmds.NewCompleted([]string{"PTTL", ck}), cmds.AskingCmd, Completed(cmd.Cmd), cmds.ExecCmd)
```

Comment:
Is the second `cmds.AskingCmd` really needed? It looks like we only need one:
<img width="940" alt="image" src="https://github.com/user-attachments/assets/e8bbbc68-e4c4-4865-ac4e-eeaf0c693c79" />
https://redis.io/docs/latest/commands/asking/

Reply:
I would run tests to find out.

Reply:
tested and yes, only one ASKING is needed. Thank you for pointing out.